containers/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+docker build -f airunner.Dockerfile -t airunner_image .
```

Comment:
all docker build files now added to the main airunner repo (originally in a private repo)